python/interpret_community/explanation/explanation.py

```diff
@@ -724,7 +724,11 @@ def data(self, key=None):
                 for local_mli_data in mli_data:
                     if local_mli_data[InterpretData.EXPLANATION_TYPE] == InterpretData.LOCAL_FEATURE_IMPORTANCE:
                         local_mli_data[InterpretData.VALUE][InterpretData.INTERCEPT] = self.expected_values
+                class_dimension = InterpretData.SINGLE
+                if PerClassMixin._does_quack(self):
+                    class_dimension = InterpretData.MULTICLASS
                 mli_data.append({
+                    InterpretData.EXPLANATION_CLASS_DIMENSION: class_dimension,
```

Comment:
IS there no more targeted way to just set this on PFI? I'm just concerned this is broader than we'd discussed

Reply:
We could have a PFIMixin

Reply:
discussed offline, this includes more than just PFI, also mimic with no eval data and possibly others in the future